lasagna-master/ena/lasagna_master.go

```diff
@@ -0,0 +1,37 @@
+package lasagna
+
+func PreparationTime(layers []string, timePerLayer int) int {
+	const averageTimePerLayer = 2
+	if timePerLayer == 0 {
+		timePerLayer = averageTimePerLayer
+	}
+	return len(layers) * timePerLayer
+}
+
+func Quantities(layers []string) (int, float64) {
+	const noodlesPerLayer = 50 // grams
+	const saucePerLayer = 0.2  // liters
```

Comment:
I get why you add the constants, but they should be outside the funcion. No point in redeclaring them with each call to this function.

Reply:
Oh, you mean globals? grrrr